src/components/dependencies/BDBARescoring.js

```diff
@@ -375,7 +380,85 @@ RescoringHeader.propTypes = {
 }
 
 
-const RescoringRowLoading = () => {
+const Filtering = ({
+  selectedSprints,
+  setSelectedSprints,
+  sprints,
+}) => {
+  const searchParamContext = React.useContext(SearchParamContext)
+  const theme = useTheme()
+
+  if (sprints.length === 0) return null
+
+  // if there are any sprints selected which don't have any associated findings, remove the selection
```

Comment:
This comment confuses me.
Code checks whether all selected sprints are valid.

Why is that required at all?
Shouldn't we only offer selection of "valid" sprints?

Reply:
Fair enough, but isn't it more intuitive to only filter "bad" sprints, rather than resetting the entire selection?

Reply:
You're probably right, my thought was that if you have a selection where some parts are "bad" (i.e. "bad" sprints defined as search param), you probably don't want this selection in place at all (as it might have been added for another scenario).